.github/workflows/release.yml

```diff
@@ -0,0 +1,159 @@
+# must do before using this action:
+# - set up RELEASE_TOKEN in secrets of the repository avocado-framework/avocado
+# - set up RTD_TOKEN in secrets to update readthedocs
+# - set up two tokens for twine: PYPI_USER and PYPI_PASSWD
+
+name: Release tests
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release version'
+        required: true
+        default: '0.0'
+      devel_name:
+        description: 'Developer Name'
+        required: true
+        default: 'Avocado Developer'
+      devel_mail:
+        description: 'Developer mail'
+        required: true
+        default: 'avocado@redhat.com'
+      rtd_project:
+        description: 'readthedocs project name'
+        required: true
+        default: 'avocado-framework'
+
+jobs:
+
+  release:
+    name: Update versions
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:34
+
+    steps:
+      - name: install required packages
+        run:  dnf -y install rpmdevtools git
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+      - name: Update VERSION files
+        run: |
+          echo ${{ github.event.inputs.version }} > VERSION
+          for DIR in optional_plugins/*; do
+                if test -f $DIR/VERSION; then
+                        echo ${{ github.event.inputs.version }} > $DIR/VERSION
+                fi
+          done
```

Comment:
What is the difference between this code and the `make propagate-version`? Based on our off-line conversation I had understood that this would be converted to something like the following to avoid duplicated code here.

```
 run : make propagate-version
```